keymirror/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for keymirror 0.1.1
+// Project: https://github.com/STRML/keyMirror
+// Definitions by: Johannes Fahrenkrug <https://github.com/jfahrenkrug>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function KeyMirror(obj: Object): Object;
```

Comment:
Could this be `declare function KeyMirror(obj: {}): { [key: string]: string }`?
Aside: Microsoft/TypeScript#12114 could help, but those aren't in `2.0` so we'll have to do the above for now.